docs/content/development/extensions-core/sample-client-sslcontext.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: doc_page
+---
+
+## Sample SSLContext Provider Module
```

Comment:
This is a "sample". What situations do we expect users to use this in? Is it expected that they'll use it in production? Is it necessary to use an extension like this in order to get TLS to work?